packages/web3-react/src/context/connectors.tsx

```diff
@@ -44,6 +44,14 @@ const ProviderConnectors: FC<ConnectorsContextProps> = (props) => {
   } = props;
 
   const { supportedChainIds } = useSDK();
+  const walletConnectRPC = useMemo(
+    () =>
+      Object.entries(rpc).reduce(
+        (result, [key, value]) => ({ ...result, [key]: BASE_URL + value }),
```

Comment:
It seems that `value` can be both an absolute and a relative url. So in the case of an absolute url it is worth to add some extra check